examples/gettingStarted/session.ts

```diff
@@ -0,0 +1,86 @@
+/*
+  This sample demonstrates how Service Bus Messages can be sent to and received from individual
+  sessions as created on session enabled queues/topic subscriptions.
+*/
+
+import {
+  SendableMessageInfo,
+  OnSessionMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage,
+  Namespace,
+  MessageSession
+} from "../../lib";
+
+// Define connection string and related Service Bus entity names here
+// Ensure on portal.azure.com that queue/topic has Sessions feature enabled
+const connectionString = "";
+const queueName = "";
+
+let ns: Namespace;
+
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(connectionString);
+  try {
+    await sendMessages("session-1");
+    await sendMessages("session-2");
+    await sendMessages("session-3");
+    await sendMessages("session-4");
+
+    await receiveMessages();
+  } finally {
+    await ns.close();
+  }
+}
+
+async function sendMessages(sessionId: string): Promise<void> {
+  // If using Topics, use createTopicClient to send to a topic
+  const client = ns.createQueueClient(queueName);
+  const data = [
+    { step: 1, title: "Shop" },
+    { step: 2, title: "Unpack" },
+    { step: 3, title: "Prepare" },
+    { step: 4, title: "Cook" },
+    { step: 5, title: "Eat" }
+  ];
```

Comment:
I believe the reason we are using steps here is so that we can then show that sessions has the FIFO feature.

I wonder if as a SDK sample, it is the responsibility of this sample to demonstrate FIFO. This uses the same argument as we used for not having the duplicateDetection sample.

The main responsibility of this sample would be just to show how to send/receive messages.

Thoughts?

If you agree, then I would fall back on using the simple scientists data that we are using in the other samples

Reply:
Hm.. agree. Since the usability study meeting, I do think it makes sense to have copy-paste-able code handy in the samples. Will simplify